bin/varnishd/proxy/cache_proxy_proto.c

```diff
@@ -692,7 +692,6 @@ VPX_Send_Proxy(int fd, int version, const struct sess *sp)
 	CHECK_OBJ_NOTNULL(sp, SESS_MAGIC);
 	assert(version == 1 || version == 2);
 	AN(VSB_new(vsb, buf, sizeof buf, VSB_FIXEDLEN));
-	AN(vsb);
```

Comment:
Please squash in 26045e48b83ba276b00ee4056581b33294b6eb49.

Reply:
will do